nutils/topology.py

```diff
@@ -1064,11 +1217,31 @@ def __init__(self, basetopo:stricttopology, vgroups:types.frozendict={}, bgroups
   def __len__(self):
     return len(self.basetopo)
 
-  def getitem(self, item):
-    if isinstance(item, str) and item in self.vgroups:
-      itemtopo = self.vgroups[item]
-      return itemtopo if isinstance(itemtopo, TransformChainsTopology) else self.basetopo[itemtopo]
-    return self.basetopo.getitem(item)
+  def get_groups(self, *groups: str) -> TransformChainsTopology:
+    topos = []
+    basegroups = []
+    for group in groups:
+      if group in self.vgroups:
+        item = self.vgroups[group]
+        assert isinstance(item, (TransformChainsTopology, str))
+        if isinstance(item, TransformChainsTopology):
+          topos.append(item)
+        else:
+          basegroups.extend(item.split(','))
+      else:
+        basegroups.append(group)
+    if basegroups:
+      topos.append(self.basetopo.get_groups(*basegroups))
+    return functools.reduce(operator.or_, topos, self.empty_like())
+
+  def take_unchecked(self, __indices: numpy.ndarray, __idim: int, __ndim: int) -> TransformChainsTopology:
+    return self.basetopo.take(__indices, __idim, __ndim)
```

Comment:
should this be take_unchecked? (idem compress and slice)

Reply:
Done.